### ISSUE TITLE
Disconnected from WSL

### ISSUE THREAD

**Issue by Manfrinne** [microsoft/vscode#136917]
Issue Type: <b>Bug</b>

Error: Disconnected from WSL:Ubunto-20.04.  Its happen several times. I use extensions to work remote on Windows WSL, but again and again its happens.

VS Code version: Code 1.62.1 (f4af3cbf5a99787542e2a30fe1fd37cd644cc31f, 2021-11-05T10:57:55.946Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No
Remote OS version: Linux x64 5.10.16.3-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-2630QM CPU @ 2.00GHz (8 x 1995)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: unavailable_off|
|Load (avg)|undefined|
|Memory (System)|7.91GB (1.97GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+Ubuntu-20.04/home/m4nfrinne/API-rentalx --remote=wsl+Ubuntu-20.04 --crash-reporter-id 3416a5f7-b6c3-415f-a552-30485dce104c|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 5.10.16.3-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i7-2630QM CPU @ 2.00GHz (8 x 1995)|
|Memory (System)|1.94GB (0.70GB free)|
|VM|0%|
</details><details><summary>Extensions (23)</summary>

Extension|Author (truncated)|Version
---|---|---
emojisense|bie|0.9.0
bracket-pair-colorizer|Coe|1.0.61
remote-containers|ms-|0.205.2
remote-ssh|ms-|0.66.1
remote-ssh-edit|ms-|0.66.1
remote-wsl|ms-|0.58.5
vscode-remote-extensionpack|ms-|0.21.0
material-icon-theme|PKi|4.10.0
markdown-emoji|bie|0.2.1
vscode-css-modules|cli|0.3.2
nunjucks-vscode-extensionpack|dou|0.0.1
gitlens|eam|11.6.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|9.0.0
vscode-env|Iro|0.1.0
vscode-styled-components|jpo|1.6.6
vscode-docker|ms-|1.17.0
color-highlight|nau|2.5.0
prisma|Pri|3.4.2
prisma-insider|Pri|31.0.286
LiveServer|rit|5.6.1
tabnine-vscode|Tab|3.4.35
markdown-emoji-review|unm|1.0.2

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwt:30382697
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis300:30386381
vsaa593cf:30376535
vssld246cf:30386378

```

</details>

**Comment by Manfrinne** [comment]
![image](https://user-images.githubusercontent.com/58799400/141222621-15d32f5a-a041-4c78-b95a-2f65701a9b67.png)

**Comment by gjsjohnmurray** [comment]
See https://github.com/microsoft/vscode/issues/136873#issuecomment-965402902 for advice on downgrading to 1.62.  If doing this fixes your issue, then wait for upcoming 1.62.2

**Comment by aeschli** [comment]
Do you still see the issue with the latest version?
Also, does it reproduce with extension disabled?
